app/src/test/kotlin/UserCurrencyTest.kt

```diff
@@ -0,0 +1,60 @@
+package com.kickstarter
+
+import com.kickstarter.libs.UserCurrency
+import com.kickstarter.mock.MockCurrentConfig
+import com.kickstarter.mock.factories.ConfigFactory
+import com.kickstarter.mock.factories.ProjectFactory
+import junit.framework.Assert
+import junit.framework.TestCase
+import type.CurrencyCode
+
+class UserCurrencyTest: TestCase() {
+
+    private fun createUserCurrency(countryCode: String): UserCurrency {
+        val config = ConfigFactory.config().toBuilder()
+                .countryCode(countryCode)
+                .build()
+
+        val currentConfig = MockCurrentConfig()
+        currentConfig.config(config)
+
+        return UserCurrency(currentConfig)
+    }
+
+    fun testFormatCurrency_withUserInUS() {
+        val currency = createUserCurrency("US")
+        Assert.assertEquals("$100", currency.format(100.0f, ProjectFactory.project(), CurrencyCode.USD.rawValue()))
+        Assert.assertEquals("CA$100", currency.format(100.0f, ProjectFactory.caProject(), CurrencyCode.CAD.rawValue()))
```

Comment:
I'm not understanding what these tests are testing. My understanding is if I prefer USD, if a project is in Canadian dollars, I'd see the conversion in USD.

For this project, I'd assume I would see US$100 because I prefer USD.

How does `UserCurrency` take in the user's currency preference?

Reply:
So for these tests it tests the user's location and based upon the location and user's preferred currency it will show the proper currency symbol. 

So for the test above the user is in the `US` and if the user's preferred currency is `$US` or they don't have a preference it still should show as `$`.

Reply:
But this test is asserting that when we call `currency.format(100.0f, ProjectFactory.caProject(), CurrencyCode.CAD.rawValue()`, it will output `"CA$100"`. Where does the user's location (`US`) come in? Shouldn't it be `US$100`?

Reply:
why does the user's location matter? 👀

Reply:
Location matters because the display of the currency like `$` or `US$` depends on the location of the user.

Reply:
ok but this test is called `testFormatCurrency_withUserInUS` so i'm assuming the user is always in the US. is the test name accurate?

Reply:
Yes isn't it? It's testing the format of the currency with the user in the US 🧐